modules/installation-osp-converting-ignition-resources.adoc

```diff
@@ -0,0 +1,111 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_openstack/installing-openstack-user.adoc
+
+[id="installation-osp-converting-ignition-resources_{context}"]
+= Preparing the bootstrap Ignition file
+
+The bootstrap Ignition file that the installation program generates is too large to pass to {rh-openstack} directly.
+
+First, upload the bootstrap Ignition configuration file. Then, create a small Ignition file ("shim") that downloads the generated ignition file to Nova as user data.
```

Comment:
a bit of confusion here.

The "shim" is the file that we pass to Nova as user data when we create the server.
The shim itself contains instructions like "hey new server, please download the real Ignition beefy file from this location: https://www.blablabla"